testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/persistence/YamlExtensionTestCase.java

```diff
@@ -66,10 +64,11 @@ public class YamlExtensionTestCase {
 
     @BeforeClass
     public static void setup() throws Exception {
+        Assume.assumeTrue("Layer testing provides a different XML file than the standard one which results in failures", System.getProperty("ts.layers") == null);
```

Comment:
Ahh, forget it, my mistake, `System.getProperty("ts.layers") == null` is the best option at the moment for this one since we need to ensure the test is not executed also if `-Dts.layers=true`